.github/workflows/superset-python-integrationtest.yml

```diff
@@ -6,8 +6,16 @@ on:
     branches:
       - "master"
       - "[0-9].[0-9]"
+    paths:
+      - "superset/**"
+      - "requirements/**"
+      - "tests/integration_tests/**"
   pull_request:
     types: [synchronize, opened, reopened, ready_for_review]
+    paths:
+      - "superset/**"
```

Comment:
I wonder if we would need to include `.pre-commit-config.yaml` or would the premise be that if `pre-commit` needed to touch any files the `superset-python-misc.yml` workflow would fail. That would then force the user run `tox -e pre-commit` locally which would then touch files in the `superset/**` and `tests/**` paths?

Reply:
`pre-commit` has its own [required] check and runs on all files (not just the last commit as it does locally) and will non-zero exit if there's anything. So that prevents people from  `commit --no-verify` or simply not setting up pre-commit